types/src/v0/impls/stake_table.rs

```diff
@@ -0,0 +1,347 @@
+use super::{L1Client, NodeState, PubKey, SeqTypes};
+use contract_bindings::permissioned_stake_table::{NodeInfo, StakersUpdatedFilter};
+use ethers::{abi::Address, types::U256};
+use hotshot::types::SignatureKey;
+use hotshot_contract_adapter::stake_table::NodeInfoJf;
+use hotshot_types::{
+    stake_table::StakeTableEntry,
+    traits::{
+        election::Membership, node_implementation::NodeType, signature_key::StakeTableEntryType,
+    },
+    PeerConfig,
+};
+use std::{
+    cmp::max,
+    collections::{BTreeMap, HashSet},
+    num::NonZeroU64,
+    str::FromStr,
+};
+use thiserror::Error;
+use url::Url;
+
+#[derive(Clone, Debug)]
+pub struct StaticCommittee<T: NodeType> {
```

Comment:
Can we avoid using generics here since we already have `SeqTypes` in this crate?

Reply:
Yes

Reply:
ef90df412caf7a0757dfc7b6387a22471136f8ff

Reply:
removed some more generics here https://github.com/EspressoSystems/espresso-sequencer/pull/2383/files